packages/ui/theme/foundations/fonts.ts

```diff
@@ -1,8 +1,8 @@
 import type { ThemeOverride } from '@chakra-ui/react';
-import primitives from '@core-ds/primitives';
+// import primitives from '@core-ds/primitives';
 
 export const fonts: ThemeOverride['fonts'] = {
-   body: primitives.fontFamily.lato,
-   heading: primitives.fontFamily.lato,
-   mono: primitives.fontFamily.monoSystem,
+   body: `-apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"`,
+   heading: `-apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"`,
+   mono: `ui-monospace, SFMono-Regular, SF Mono, Menlo,Consolas, Liberation Mono, monospace`,
```

Comment:
Doesn't look like `ui-monospace` is well supported: https://caniuse.com/?search=ui-monospace